tests/rules/test_json_schema.py

```diff
@@ -6,13 +6,32 @@
 
 
 tags_inputs = [
-    (["id"], [], {"category": ["id"]}, {Level.INFO: [("category",)]}),
+    (
```

Comment:
What do you think of the following?

* A test case with no tag
* A test case with all tags